.github/workflows/build.yml

```diff
@@ -90,24 +90,40 @@ jobs:
           - test-type: pytype
             python-version: 3.8
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v1
+      id: setup_python
+      uses: actions/setup-python@v4
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
+    - name: Get week and year
+      id: week_year
+      run: echo "DATE=$(date +%U-%Y)" >> $GITHUB_OUTPUT
+    - name: Cached virtual environment
+      id: venv_cache
+      uses: actions/cache@v3
+      with:
+        path: venv
+        key: pip-${{ steps.setup_python.outputs.python-version }}-${{ steps.week_year.outputs.DATE }}-${{ hashFiles('**/requirements.txt', 'setup.py') }}
```

Comment:
What's the recommended way to launch a Github build that forces an uncached environment?
(This might be useful to test dependency interplay at HEAD).

Reply:
You can just delete the caches from the Actions submenu.  We could also probably figure out some way to signal against using caches, but in practice just deleting them is probably enough.